src/orquestra/integrations/qiskit/runner/_ibmq_runner.py

```diff
@@ -41,16 +45,7 @@ def create_ibmq_runner(
     retry_timeout_seconds: int = 24 * 60 * 60,  # default timeout of one day
     discard_extra_measurements=False,
 ):
-    try:
```

Comment:
Did you delete this try except statement because IBM now provides an easily legible error?